index.html

```diff
@@ -1661,6 +1671,17 @@ <h2 id="AudioContext">
           use cases, only a single <a><code>AudioContext</code></a> is used per
           document.
         </p>
+        <p>
+          An <a><code>AudioContext</code></a> is said to be <dfn>allowed to
+          start</dfn> if the user agent and the system allow sound output in
```

Comment:
I'm not sure how much I can say. This is a normative text that describes what `allowed to start` means. The note tries to give some more colour but it sounds fair for the reader to search in the spec how this is being used.